docs/celo-codebase/protocol/bridging/optics-bridge-faq.md

```diff
@@ -54,3 +70,113 @@ Some tokens are not currently supported (Polygon WETH, Polygon SUSHI, etc)
 ## My newly-added token has an ugly name, can it be changed?
 
 Yes, reach out to us on [#bridge-support](https://discord.gg/Rp8TYetc) and we’ll request a name change.
+
+
+## I sent USDC to Polygon but I think I received the wrong token, how do I resolve? 
+
+If you obtained USDC tokens which are not recognized on the Polygon network, you might have sent the wrong token. There are two core USDC token contracts, one that is native to the Ethereum network and the another that represents USDC on the Polygon network. There are multiple USDC contracts on different networks, each corresponding to the bridge that was used to send the token. For example, with Polygon, USDC sent via the [Polygon Bridge](https://wallet.polygon.technology/) will have a different address than USDC sent via the Optics Bridge (and it doesn’t matter if the USDC is comes from Celo or directly from Ethereum). On Polygon, there are greater incentives for users to bridge USDC via the polygon bridge, so that is the more popular token with greater liquidity.  To transfer the popular version of Polygon USDC from Celo to Polygon via Optics, make sure that you use USDC PoS. In the event you sent the wrong token, bridge back the tokens to convert to USDC PoS and then bridge USDC PoS to Polygon.
+
+
+## How do I transfer value from Celo to Polygon? 
+
+It’s recommended to use USDC PoS.
+
+To get USDC using token with address 0x2791bca1f2de4661ed88a30c99a7a9449aa84174 on Polygon you should send USDC PoS with token address 0x1bfc26cE035c368503fAE319Cc2596716428ca44 from Celo.
+
+
+## How do I verify token addresses? 
+
+Within the Optics.app dashboard, on the top right side of the page, select your wallet address to bring up the contract validation window to view token addresses.
+
+
+After connecting your wallet, tap your wallet address:
+<ImageWrapper path="/img/doc-images/migrating-optics-v2/verify-token-wallet.png" alt="Connect your wallet" width="400" />
+
+This will display the list of tokens:
+<ImageWrapper path="/img/doc-images/migrating-optics-v2/verify-token-list.png" alt="View token list" width="500" />
+
+
+
+## Resources
+
+
+### Dashboards
```

Comment:
The rendering of h3 is showing up as h2. We need to update the staying of these pages to accommodate the difference. Otherwise, this looks great!